packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -41,6 +41,16 @@ function backtickIfNeeded(e: string): string | undefined {
   }
 }
 
+function backtickDbNameIfNeeded(e: string): string | undefined {
```

Comment:
We could skip this separate method and expand "backtickIfNeeded" with an option-parameter that decides which symbols we should allow. Let me know if you have a preference

Reply:
With a parameter to check what regex to use then, the difference between the methods is that this method allows for "." in the text without adding backticks